src/libraries/System.Net.Http/src/System/Net/Http/Headers/HttpHeaderValueCollection.cs

```diff
@@ -165,26 +165,25 @@ public bool Remove(T item)
         public IEnumerator<T> GetEnumerator()
         {
             object? storeValue = _store.GetParsedValues(_descriptor);
+            return storeValue is null ?
+                ((IEnumerable<T>)Array.Empty<T>()).GetEnumerator() :
```

Comment:
Add comment? (I assume Array.Empty uses a singleton for its enumerator, but it's not obvious...)